pyhomematic/devicetypes/thermostats.py

```diff
@@ -162,13 +162,29 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         super().__init__(device_description, proxy, resolveparamsets)
 
         # init metadata
-        self.SENSORNODE.update({"ACTUAL_TEMPERATURE": [1]})
+        self.SENSORNODE.update({"ACTUAL_TEMPERATURE": [1],
+                                "VALVE_STATE": [1]})
```

Comment:
Other Thermostats have valve as attribute node. So you can use it with a template sensor on hass. I think we should that handle in same way.